src/core_atmosphere/physics/physics_wrf/module_cu_ntiedtke.F

```diff
@@ -210,3667 +83,285 @@ subroutine cu_ntiedtke(                                    &
+!                      con_xls = xls , con_xlf = xlf , con_grav = grav , errmsg  = errmgs , &
+!                      errflg  = errflg                                                     &
+!                     )
+
+
+ do j = jts,jte
+    do i = its,ite
+       cu_act_flag(i,j)=.true.
+    enddo
+ enddo
+
+ do j = jts,jte
+
+    do i = its,ite
+       dx_hv(i)    = dx(i,j)
+       hfx_hv(i)   = hfx(i,j)
+       qfx_hv(i)   = qfx(i,j)
+       xland_hv(i) = xland(i,j)
+    enddo
+
+    do k = kts,kte
+       do i = its,ite
+          dz_hv(i,k)   = dz8w(i,k,j)
+          pi_hv(i,k)   = pi3d(i,k,j)
+          prsl_hv(i,k) = pcps(i,k,j)
+          qv_hv(i,k)   = qv3d(i,k,j)
+          qc_hv(i,k)   = qc3d(i,k,j)
+          qi_hv(i,k)   = qi3d(i,k,j)
+          rho_hv(i,k)  = rho3d(i,k,j)
+          t_hv(i,k)    = t3d(i,k,j)
+          u_hv(i,k)    = u3d(i,k,j)
+          v_hv(i,k)    = v3d(i,k,j)
+
+          qvften_hv(i,k) = qvften(i,k,j)
+          thften_hv(i,k) = thften(i,k,j)
+       enddo
+    enddo
+    do k = kts,kte+1
+       do i = its,ite
+          prsi_hv(i,k) = p8w(i,k,j)
+          w_hv(i,k)    = w(i,k,j)
+       enddo
+    enddo
+
+!   call cu_ntiedtke_timestep_init(its,ite,kts,kte,im,kx,kx1,itimestep,stepcu,dt,grav,xland_hv,dz_hv, &
+!                   prsl_hv,prsi_hv,t_hv,rho_hv,qv_hv,qc_hv,qi_hv,u_hv,v_hv,w_hv,qvften_hv,thften_hv, &
+!                   qvftenz,thftenz,slimsk,delt,prsl,ghtl,tf,qvf,qcf,qif,uf,vf,prsi,ghti,omg,         &
+!                   errmsg,errflg)
+    call cu_ntiedtke_timestep_init( &
+                     its    = its       , ite     = ite     , kts     = kts     , kte       = kte       , &
+                     im     = im        , kx      = kx      , kx1     = kx1     , itimestep = itimestep , &
+                     stepcu = stepcu    , dt      = dt      , grav    = grav    , xland     = xland_hv  , &
+                     dz     = dz_hv     , pres    = prsl_hv , presi   = prsi_hv , t         = t_hv      , &
+                     rho    = rho_hv    , qv      = qv_hv   , qc      = qc_hv   , qi        = qi_hv     , &
+                     u      = u_hv      , v       = v_hv    , w       = w_hv    , qvften    = qvften_hv , &
+                     thften = thften_hv , qvftenz = qvftenz , thftenz = thftenz , slimsk    = slimsk    , &
+                     delt   = delt      , prsl    = prsl    , ghtl    = ghtl    , tf        = tf        , &
+                     qvf    = qvf       , qcf     = qcf     , qif     = qif     , uf        = uf        , &
+                     vf     = vf        , prsi    = prsi    , ghti    = ghti    , omg       = omg       , &
+                     errmsg = errmsg    , errflg  = errflg                                                &
+                                  )
+
+!   call cu_ntiedtke_run(uf,vf,tf,qvf,qcf,qif,qvftenz,thftenz,ghtl,ghti,omg,prsl,prsi,qfx_hv,hfx_hv, &
+!                        rn,slimsk,im,kx,kx1,delt,dx_hv,errmsg,errflg)
```

Comment:
@ldfowler58 Why do you want to keep this (and other calls) commented out, rather than removed?

Reply:
as you suggested, I removed commented out calls to different subroutines in cu_ntiedtke.F